src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class Truck {
```

Comment:
The class Truck is empty. Ensure that it extends the Machine class and implements any necessary methods, such as start() and stop(), with informative messages indicating the type of Machine.